src/installer/installer.go

```diff
@@ -96,6 +98,19 @@ func (i *installer) FormatDisks() {
 	}
 }
 
+func (i *installer) StorePublicKeyInControlPlane(publicKey []byte, client k8s_client.K8SClient) error {
+	err := client.CreateNamespace("node-public-keys")
```

Comment:
Prefix with `assisted-` so its obvious this is an assisted mechanism and not an OCP thing